core/src/main/java/com/linecorp/armeria/common/util/EventLoopThreadFactory.java

```diff
@@ -19,48 +19,55 @@
 import static java.util.Objects.requireNonNull;
 
 import java.util.concurrent.ThreadFactory;
+import java.util.function.Function;
 
 import javax.annotation.Nullable;
 
-import io.netty.util.concurrent.DefaultThreadFactory;
-
 /**
  * {@link ThreadFactory} that creates event loop threads.
  *
  * @see EventLoopGroups
+ *
+ * @deprecated Use {@link ThreadFactories#newEventLoopThreadFactory(String, boolean)} or
+ * {@link ThreadFactories#builder(String)}. Note that setting
+ * {@link ThreadFactoryBuilder#eventLoop(boolean)} true is required to create event loop thread. e.g.
```

Comment:
- `it is required to set {@code true} to {@link ...#eventLoop(boolean)}` to create event loop threads, e.g.`
  - Note the comma between the sentence and 'e.g.'
  - thread -> threads

Reply:
Fixed typos. Thank you.